compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/KnownTargets.h

```diff
@@ -12,6 +12,10 @@
 
 namespace mlir::iree_compiler::IREE::GPU {
 
+static constexpr char kNoEncodingLayoutResolverName[] = "none";
+static constexpr char kPadEncodingLayoutResolverName[] = "pad";
+static constexpr char kDataTilingEncodingLayoutResolverName[] = "data-tiling";
```

Comment:
nit: static is redundant here: `constexpr` in the global/namespace scope already makes this static.